### PR TITLE
COMMERCE-832 Make availability dots function properly

### PR DESCRIPTION
**Pull request by alecsloan**
Let me know if you have any questions or changes.

### CODE REVIEW

**Comment by ethanbustad** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:retest

**Comment by marco-leo** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
Merged into `7.1.x`, which we'll rebase `7.1.x-next` on top of.

**Comment by ethanbustad** [comment]
Sorry, I lied, Marco wanted changes, so we didn't merge this one.

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
@marco-leo This PR is ready for re-review. As the changes you requested/we discussed have been pushed

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

File: commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/product/renderer/list/MiniumCPContentListRenderer.java [hunk 1]
```diff
@@ -66,14 +76,55 @@ public void render(
 			HttpServletResponse httpServletResponse)
 		throws Exception {
 
+		CPDataSourceResult cpDataSourceResult =
+			(CPDataSourceResult)httpServletRequest.getAttribute(
+				CPWebKeys.CP_DATA_SOURCE_RESULT);
+
+		List<CPSku> cpSkus;
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/product/renderer/list/MiniumCPContentListRenderer.java:83]: initialize `List<CPSku> cpSkus` to `null`.

**Review by ethanbustad — COMMENTED** [review]
@alecsloan just one change here

**Pull request merged**